source/agora/node/FullNode.d

```diff
@@ -521,9 +521,18 @@ public class FullNode : API
     ***************************************************************************/
 
     protected void onAcceptedBlock (const ref Block block,
-        bool validators_changed) @safe
+        bool validators_changed) @safe nothrow
     {
-        this.pushBlock(block);
+        try
+        {
+            this.pushBlock(block);
+        }
+        catch (Exception ex)
+        {
+            log.error("Error pushing block: {}" ~
+                "Block: {}", ex.msg, block);
+            assert(0);
```

Comment:
So `pushBlock` can never `throw` ?

Reply:
Since `pushBlock` is pushing to a handler, it should be recoverable.
Therefore, `assert(0)` is not the right solution.